Src/WitsmlExplorer.Frontend/components/Modals/DownloadOptionsSelectionModal.tsx

```diff
@@ -0,0 +1,251 @@
+  ): [(row: any) => any, string] => {
+    let comparator;
+    switch (column.type) {
+      case ContentType.Number:
+        comparator = (row: any): number => Number(row[column.property]);
+        break;
+      case ContentType.Measure:
+        comparator = (row: any): number =>
+          Number(indexToNumber(row[column.property]));
+        break;
+      default:
+        comparator = (row: any): string => row[column.property];
+        break;
+    }
+    return [comparator, column.property];
+  };
+
+  const exportSelectedDataPoints = useCallback(() => {
+    const exportColumns = props.columns
+      .map((column) => `${column.columnOf.mnemonic}[${column.columnOf.unit}]`)
+      .join(exportOptions.separator);
+    const data = orderBy(
+      props.selectedRows,
+      getComparatorByColumn(props.columns[0]),
+      [Order.Ascending, Order.Ascending]
+    ) //Sorted because order is important when importing data
+      .map((row) =>
+        props.columns
+          .map((col) => row[col.columnOf.mnemonic] as string)
+          .join(exportOptions.separator)
+      )
+      .join(exportOptions.newLineCharacter);
+    exportData(props.log.name, exportColumns, data);
+  }, [props.columns, props.selectedRows]);
+
+  const executeExport = () => {
+    switch (downloadOptions) {
+      case DownloadOptions.All:
+        exportAll();
+        break;
+      case DownloadOptions.SelectedRange:
+        exportSelectedRange();
+        break;
+      case DownloadOptions.SelectedIndexValues:
+        exportSelectedDataPoints();
+    }
+    downloadOptions = DownloadOptions.SelectedRange;
+  };
+
+  return (
+    <ConfirmModal
+      heading={`Download log data for ${props.mnemonics.length} mnemonics`}
+      content={
+        <>
+          <span>
+            <Typography>Choose download option?</Typography>
+          </span>
+
+          <label style={alignLayout}>
+            <Radio
+              name="group"
+              value={DownloadOptions.SelectedRange}
+              id={DownloadOptions.SelectedRange}
+              onChange={onChangeDownloadOption}
+              defaultChecked
+            />
+            <Typography>Download selected range</Typography>
+          </label>
+          <label style={alignLayout}>
+            <Radio
+              name="group"
+              id={DownloadOptions.SelectedIndexValues}
+              value={DownloadOptions.SelectedIndexValues}
+              onChange={onChangeDownloadOption}
+              disabled={!props.selectedRows.length}
+            />
+            <Typography>Download selected rows</Typography>
+          </label>
+          <label style={alignLayout}>
+            <Radio
+              name="group"
+              id={DownloadOptions.All}
+              value={DownloadOptions.All}
+              onChange={onChangeDownloadOption}
+            />
+            <Typography>Download all data</Typography>
+          </label>
+          {visibleFileTypeSelection && (
```

Comment:
Instead of hiding it entirely, does it make more sense to disable the LAS option? That way the user would know what file type to expect.